api/src/test/java/org/pmiops/workbench/interceptors/AuthInterceptorTest.java

```diff
@@ -47,95 +60,114 @@ public void handle() {}
   public void handle() {}
 }
 
+@RunWith(SpringRunner.class)
```

Comment:
A lot of the changes in this file were related to Spring-ifying this test class, and pulling some very common logic into simple helper methods.